extensions/keycodes/internal.m

```diff
@@ -168,6 +169,12 @@ int keycodes_cachemap(lua_State* L) {
     pushkeycode(L, kVK_DownArrow, "down");
     pushkeycode(L, kVK_UpArrow, "up");
 
+    pushkeycode(L, kVK_JIS_Yen, "¥");
+    pushkeycode(L, kVK_JIS_Underscore, "underscore");
+    pushkeycode(L, kVK_JIS_KeypadComma, "padcomma");
```

Comment:
Is there a reason to label this "padcomma" instead of "pad,"?  Other typographical's on the keypad are done with the character itself and not the name (e.g. "pad.")